source/Main.py

```diff
@@ -183,17 +179,39 @@ def get(self):
             'login_text': login_text,
             'app': app_identity.get_application_id()}
 
+        self.response.content_type = 'text/html'
+        path = os.path.join(os.path.dirname(__file__), '../templates/Create.html')
+        self.response.write(template.render(path, template_values))
+
+
+class ViewPage(webapp2.RequestHandler):
```

Comment:
Could we change this class name to look more like `ViewAllPage` or `ViewAllStreamsPage` or something? Just so it doesn't get confused with the page for viewing a single stream.